testing/src/main/java/org/oppia/testing/TestFirebaseModule.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.testing
+
+import dagger.Module
+import dagger.Provides
+import java.lang.Exception
+import javax.inject.Singleton
+import org.oppia.util.firebase.CrashLogger
+
+@Module
+class TestFirebaseModule {
```

Comment:
TestLogReportingModule maybe? (Depends on the name you use for the module below)

Reply:
Done